### PR TITLE
MES-4311 -  Vehicle Dimensions & UX Review Changes

### PR DESCRIPTION
**Pull request by TomBillingtonUK**
## Description
- Updated app to use the latest schema
- Refactored the Vehicle Details card and added the vehicle details row.
- Fixed issue with text size in the Display Address common component
- Removed the uncouple / recouple row from the debrief card and added it to the test requirements row
- Fixed incorrect copy on the Vehicle checks and Uncouple / recouple faults
- Fixed spacing issues when displaying the vehicle checks questions on the debrief card
- Fixed issues with the Additional Details row on the Test Details card
- Corrected the page title so it reads Test information
- Fixed alignment issue on activity code on the search result slot

## Checklist

- [x] PR title includes the JIRA ticket number
- [x] Branch is rebased against the latest develop
- [x] Code has been tested manually
- [x] PR link added to JIRA ticket
- [ ] One review from each scrum team
- [ ] Squashed commit contains the JIRA ticket number

## Screenshots (optional)

Lots of screens for this one

Simulator Screen Shot - iPad Pro (10 5-inch) - 2019-12-12 at 22 29 24

Simulator Screen Shot - iPad Pro (10 5-inch) - 2019-12-12 at 22 29 34

Simulator Screen Shot - iPad Pro (10 5-inch) - 2019-12-12 at 22 29 45

Simulator Screen Shot - iPad Pro (10 5-inch) - 2019-12-12 at 22 29 52

Simulator Screen Shot - iPad Pro (10 5-inch) - 2019-12-12 at 22 30 01

Simulator Screen Shot - iPad Pro (10 5-inch) - 2019-12-12 at 22 30 09

Simulator Screen Shot - iPad Pro (10 5-inch) - 2019-12-12 at 22 30 18

Simulator Screen Shot - iPad Pro (10 5-inch) - 2019-12-12 at 22 30 23

### CODE REVIEW

File: src/pages/test-results-search/components/search-result/search-result.html [hunk 1]
```diff
@@ -18,12 +18,12 @@ <h3>{{getName()}}</h3>
       <ion-col col-8 no-padding>
         <h2>{{searchResult.category}}</h2>
       </ion-col>
-      <ion-col col-16 no-padding>
-        <h2>{{searchResult.activityCode}}</h2>
+      <ion-col col-16 no-padding padding-right>
```
Comment (rossfauldsbjss) [inline src/pages/test-results-search/components/search-result/search-result.html:21]: no-padding and padding-right?

File: src/shared/models/competency-display-name.ts [hunk 2]
```diff
@@ -2,6 +2,6 @@ export enum CompetencyDisplayName {
   CONTROLLED_STOP = 'Controlled Stop',
   EYESIGHT_TEST = 'Eyesight Test',
   SHOW_ME_TELL_ME = 'Show Me/Tell Me',
-  VEHICLE_CHECKS = 'Vehicle Checks',
-  UNCOUPLE_RECOUPLE = 'Uncouple Recouple',
+  VEHICLE_CHECKS = 'Vehicle checks',
+  UNCOUPLE_RECOUPLE = 'Uncouple/ recouple',
```
Comment (rossfauldsbjss) [inline src/shared/models/competency-display-name.ts:6]: Could do with some consistency on / - show me tell me has no spaces.  Uncouple recouple has one space after (which looks odd)

**Review by rossfauldsbjss — APPROVED** [review]
looks good

File: src/shared/models/competency-display-name.ts (hunk 2 above)
Comment (TomBillingtonUK) [reply src/shared/models/competency-display-name.ts:6]: will ask UX what they want, nice spot :)

File: src/pages/test-results-search/components/search-result/search-result.html (hunk 1 above)
Comment (TomBillingtonUK) [reply src/pages/test-results-search/components/search-result/search-result.html:21]: It's something we do on all the test slots removes the left , top and bottom padding but keeps thr right

**Review by ammarhaiderbjss — APPROVED** [review]

**Pull request merged**